### PR TITLE
CASMPET-6148 & CASMPET-6162 Changes from papaya shutdown test on shandy

### PR DESCRIPTION
**Pull request by bklei**
Covers CASMPET-6148 and CASMPET-6162
TBD : backport the release/1.3 changes to main

### CODE REVIEW

**Review by cdelatte-hpe — APPROVED** [review]

File: operations/power_management/Prepare_the_System_for_Power_Off.md [hunk 1]
```diff
@@ -332,6 +359,12 @@ An authentication token is required to access the API gateway and to use the `sa
 
 1.  Follow the vendor workload manager documentation to drain processes running on compute nodes. For Slurm, see the `scontrol` man page. For PBS Professional, see the `pbsnodes` man page.
 
+    Below is an example of how to drain nodes using `slurm`. The list of nodes can be copy/pasted from the `sinfo` command for nodes in an `idle` state:
+
+    ```bash
+    scontrol update NodeName=scontrol update NodeName=nid[001001-001003,001005] State=DRAIN Reason="Shutdown" State=DRAIN Reason=<reason>
```
Comment (rgross727) [inline operations/power_management/Prepare_the_System_for_Power_Off.md:365]: scontrol update NodeName=nid[001001-001003,001005] State=DRAIN Reason="Shutdown"
scontrol update NodeName=ALL State=DRAIN Reason="Shutdown"

File: operations/power_management/System_Power_Off_Procedures.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ To save management switch configuration settings, refer to [Save Management Netw
 
 ## Power Off System Cabinets
 
-To power off standard rack and liquid-cooled cabinet PDUs, refer to [Power Off Compute and IO Cabinets](Power_Off_Compute_and_IO_Cabinets.md).
+To power off standard rack and liquid-cooled cabinet PDUs, refer to [Power Off Compute Cabinets](Power_Off_Compute_Cabinets.md).
```
Comment (rgross727) [inline operations/power_management/System_Power_Off_Procedures.md:25]: This link does not work. Maybe you need to delete the old file, Power_Off_Compute_and_IO_Cabinets.md
and have a new file named Power_Off_Compute_Cabinets.md?
Or maybe move it to its new name?

**Review by rgross727 — CHANGES_REQUESTED** [review]

**Comment by rgross727** [comment]
In the Power_On_and_Start_the_Management_Kubernetes_Cluster.md document there are already several `cronjob`s which when rendered look different: "cronjob s" Not sure why?

File: operations/README.md [hunk 3]
```diff
@@ -139,7 +139,7 @@ Additional links to power off sub-procedures provided for reference. Refer to th
 - [Prepare the System for Power Off](power_management/Prepare_the_System_for_Power_Off.md)
 - [Shut Down and Power Off Compute and User Access Nodes](power_management/Shut_Down_and_Power_Off_Compute_and_User_Access_Nodes.md)
 - [Save Management Network Switch Configuration Settings](power_management/Save_Management_Network_Switch_Configurations.md)
-- [Power Off Compute and IO Cabinets](power_management/Power_Off_Compute_and_IO_Cabinets.md)
+- [Power Off Compute Cabinets](power_management/Power_Off_Compute_Cabinets.md)
```
Comment (rgross727) [inline operations/README.md:142]: This link does not work. Need to move the old file, Power_Off_Compute_and_IO_Cabinets.md
to its new name: Power_Off_Compute_Cabinets.md and edit the title in the moved file.

**Review by rgross727 — APPROVED** [review]

**Review by bo-quan — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

File: operations/power_management/Power_On_Compute_Cabinets.md [hunk 4]
```diff
@@ -43,8 +43,12 @@ power-on command from Cray System Management \(CSM\) software.
 
 1. Use the System Admin Toolkit \(`sat`\) to power on liquid-cooled cabinets chassis and slots.
 
+   Note that the default timeout for powering the compute cabinets is 60 seconds, which is generally too low.
```
Comment (haasken-hpe) [inline operations/power_management/Power_On_Compute_Cabinets.md:46]: @bklei, was this added due to CRAYSAT-1623? If so, it is not going to help with the timeout that occurs while waiting for the hms-discovery cronjob to be scheduled. I'm going to remove it when I rebase my PR #2841 on the latest main branch.

**Pull request merged**